spec/models/menu_spec.rb

```diff
@@ -0,0 +1,33 @@
+require "rails_helper"
+
+RSpec.describe Menu, type: :model do
+  before {
+    allow(Rails).to receive(:root).and_return(file_fixture("_pages").join("..").cleanpath)
+  }
+
+  subject {
+    pages = Page.build_hierarchy(file_fixture("_pages"))
+    described_class.load(
+      file_fixture("_settings/navigation.yml").cleanpath,
+      pages
+    )
+  }
```

Comment:
[suggestion, nonblocking] Rather than do the setup all in the subject block, it'd be more conventional to pull `pages` into its own thing like this:

```suggestion
  let(:pages) { Page.build_hierarchy(file_fixture("_pages")) }
  subject {
    described_class.load(
      file_fixture("_settings/navigation.yml").cleanpath,
      pages
    )
  }
```